### PR TITLE
24667 - fix missing body for filing when bootstrap business

### PR DESCRIPTION
**Pull request by hfekete**
*Issue:24667 *https://github.com/bcgov/entity/issues/24667

*Description of changes:*
fix issue when no body shown for bootstrap filing when filing is completed (on bootstrap business screen)

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

**Comment by hfekete** [comment]
for proper test, needs validate PR in first

**Review by severinbeauvais — COMMENTED** [review]
Can you please manually verify an amalgamation/continuation/incorporation/registration and show the results?

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-108-z69vntif.web.app

File: src/utils/navigate.ts [hunk 1]
```diff
@@ -1,7 +1,9 @@
...
-export const reloadWithBusinessId = () => {
+export const reloadWithBusinessId = (filing: ApiResponseFilingI) => {
   // build the URL to the business dashboard with the business id and any URL parameters
   // const url = getDashboardUrl + filing.businessIdentifier // + this.$route.fullPath
-  const url = window.location.href
-  window.location.assign(url)
+  const businessIdentifier = filing.businessIdentifier
+  const url = new URL(window.location.href)
+  url.pathname = `/${businessIdentifier}`
+  window.location.assign(url.toString())
```
Comment (severinbeauvais) [inline src/utils/navigate.ts:8]: How can I test this?

**Comment by severinbeauvais** [comment]
IA test result:

image

File: src/utils/navigate.ts (hunk 1 above)
Comment (hfekete) [reply src/utils/navigate.ts:8]: waiting for Patricks PR to get merged in

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-108-z69vntif.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-108-z69vntif.web.app

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://business-dashboard-dev--pr-108-z69vntif.web.app

**Comment by hfekete** [comment]
Registration example:

https://business-dashboard-dev--pr-108-z69vntif.web.app/TykAVmDPa7?filing_id=154838&status=Q09NUExFVEVE&accountid=2995

image

**Comment by hfekete** [comment]
Continuation-in example (thank you Patrick !!)

https://business-dashboard-dev--pr-108-z69vntif.web.app/TTk5dZdqlE
image

**Comment by hfekete** [comment]
will add amalgamation bit later

**Comment by hfekete** [comment]
Amalgamation example:
https://dev.business.bcregistry.gov.bc.ca/T73Gf87ZVQ?filing_id=154893&status=Q09NUExFVEVE

image

**Review by severinbeauvais — APPROVED** [review]

**Pull request merged**